lld/test/ELF/dtlto/archive-thin.test

```diff
@@ -29,9 +29,9 @@ RUN: mkdir %t/out && cd %t/out
 ## received JSON, pretty-prints the JSON and the supplied arguments, and then
 ## exits with an error. This allows FileCheck directives to verify the
 ## distributor inputs.
-RUN: echo '%t/t1.a %t/lib/t2.a ../t3.a \
-RUN:   --thinlto-distributor="%python" \
-RUN:   --thinlto-distributor-arg="%llvm_src_root/utils/dtlto/validate.py"' > rsp
+RUN: echo "%t/t1.a %t/lib/t2.a ../t3.a \
```

Comment:
This is ok , but the idiomatic way is to use outer single quotes and inner double quotes

Reply:
Agreed. I'll make that work and then adjust all the tests - I have used this pattern in a few places.